frontend/src/Dashboard/CrossingUpdatesPage/CrossingUpdates.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import CrossingMap from '../../Map/CrossingMap'; 
+import './CrossingUpdates.css';
+
+class CrossingUpdates extends Component {
+  state = {
+    activeTab: 'list'
+  }
+
+  changeTab(tab) {
+    this.setState({activeTab: tab});
+  }
+
+  render() {
+    return (
+      <div className="CrossingUpdates">
+        <div className="flexcontainer">
+          <div className="CrossingUpdates__tab--header">SHOW: </div>
+          <div className={this.state.activeTab === 'list' ? 'CrossingUpdates__tab--active' : 'CrossingUpdates__tab--inactive'}
+               onClick={this.changeTab.bind(this,'list')}
+          >LIST</div>
+          <div className="CrossingUpdates__tab--header"> | </div>
+          <div className={this.state.activeTab === 'map' ? 'CrossingUpdates__tab--active' : 'CrossingUpdates__tab--inactive'}
+               onClick={this.changeTab.bind(this,'map')} 
+          >MAP</div>
+        </div>
+        {this.state.activeTab === 'map' ? 
+          <div className="CrossingUpdates__map-container">
+            <CrossingMap mapHeight="80vh" mapWidth="80vw"/>
```

Comment:
When I was playing around in this branch locally, I noticed that the Map seems to rerender each time you toggle back and forth. Probably not something to over-optimize at this point, but maybe instead of removing it from the DOM completely, we could add/remove a `hidden` class when we toggle.

Reply:
Definitely worth looking into, not sure if it makes more sense to render the hidden map when going to the list view (maybe that gets slowed down).

We could probably keep track of the rendered state of the map, render when switching to the map first, then hide if switching back.